specs-go/config.go

```diff
@@ -406,6 +408,58 @@ type Anet struct {
 	Macaddress string `json:"macAddress,omitempty"`
 }
 
+// Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
+type Windows struct {
```

Comment:
Oops, meant to update the Windows one. Reverted this and fixed the Windows one instead in d57e63f35e268110de046d50ae84157603e427da